### PR TITLE
feat: add vue devtools extension

### PR DESCRIPTION
**Pull request by u3u**

### CODE REVIEW

File: generator/index.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ module.exports = api => {
       'serve:electron': 'vue-cli-service serve:electron'
     },
     devDependencies: {
-      electron: '^2.0.2'
+      electron: '^2.0.2',
+      'electron-devtools-installer': '^2.2.4'
```
Comment (TotomInc) [inline generator/index.js]: Why is the dependency name between **''**?

**Comment by nklayman** [comment]
When I try this, I run into [this issue](https://github.com/MarshallOfSound/electron-devtools-installer/issues/55). What platform are you using? I'm on windows.

**Comment by nklayman** [comment]
Thanks for the suggestion. I had to clone electron-devtools-installer and modify to fix the aforementioned issue. I also stripped away all the non-vue parts, and made it work with promises. With beta.6, this functionality is available.

**Comment by TotomInc** [comment]
@nklayman isn't it better to make a PR in `electron-devtools-installer` repo then implement vue devtools from `electron-devtools-installer`? We will benefit from their updates which most likely includes bug fixes and maybe new features.

**Comment by nklayman** [comment]
I could, but there isn't a whole lot of development going on, so that isn't a huge issue. Also, I removed the non-vue components to reduce code size. I will look into to making a PR anyway as it will fix the 7zip issue.

**Comment by motin** [comment]
I added a possible fix in this PR: https://github.com/MarshallOfSound/electron-devtools-installer/pull/92 - based on what I think was the fix mentioned in https://github.com/nklayman/vue-cli-plugin-electron-builder/pull/35#issuecomment-407164967

**Pull request closed without merging**